src/components/admin/OrdersManager.tsx

```
@@ -1,242 +1,285 @@
+    let matchesDate = true;
+    if (dateFilter !== 'all') {
+      const orderDate = new Date(order.timestamp);
+      const now = new Date();
+      
+      switch (dateFilter) {
+        case 'today':
+          matchesDate = orderDate.toDateString() === now.toDateString();
+          break;
+        case 'week':
+          const weekAgo = new Date(now.getTime() - 7 * 24 * 60 * 60 * 1000);
+          matchesDate = orderDate >= weekAgo;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)